jadx-core/src/main/java/jadx/core/utils/files/ZipSecurity.java

```diff
@@ -14,14 +15,24 @@
 
 	private ZipSecurity() {}
 
-	private static boolean isInSubDirectory(File base, File file) {
-		if (file == null) {
+	private static boolean isInSubDirectoryInternal(File baseDir, File canonFile) {
+		if (canonFile == null) {
 			return false;
 		}
-		if (file.equals(base)) {
+		if (canonFile.equals(baseDir)) {
 			return true;
 		}
-		return isInSubDirectory(base, file.getParentFile());
+		return isInSubDirectoryInternal(baseDir, canonFile.getParentFile());
+	}
+	
+	public static boolean isInSubDirectory(File baseDir, File file) {
```

Comment:
Please convert baseDir to absolute path too. Otherwise, jadx skips all resources if run on file without full path like: `jadx some.apk`.

Reply:
I changed method ``` isInSubDirectory ```, but didn't change ``` isInSubDirectoryInternal ```, because it's used only for canonical paths in the both calls